content/en/platform/corda/5.0/key-concepts/cluster-admin/virtual-nodes/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Virtual Nodes"
+date: 2023-07-24
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-cluster-admin-vnodes
+    parent: corda5-key-concepts-cluster-admin
+    weight: 3000
+section_menu: corda5
+---
+
+# Virtual Nodes
+
+From Corda 5 onwards, a single Corda deployment can host multiple virtual node identities. Previous to this, the identity and the compute process/JVM were tied. Hence in this version of Corda, we refer to virtual nodes, rather than nodes. They are now a context in which a distributed application runs under rather than the process itself.
```

Comment:
```suggestion
From Corda 5 onwards, a single Corda deployment can host multiple virtual node identities. Previous to this, the identity and the compute process/JVM were tied. Hence in this version of Corda, we refer to virtual nodes, rather than nodes. They are now a context in which a distributed application runs, rather than the process itself.
```